ups/kht.table

```diff
@@ -0,0 +1,3 @@
+setupRequired(eigen)
```

Comment:
Have you tried building this package on Jenkins? I didn't think that `eigen` is included in EUPS anymore, since @brianv0 migrated `eigen` to `conda`, so I would think that this would fail with the current stack.

Reply:
Apparently `eigen` is a special case. I actually had to add this line to get the code to build  on jenkins.

Reply:
¯\_(ツ)_/¯